package.json

```diff
@@ -7,92 +7,69 @@
+    "lint:ts": "pnpm turbo lint:ts",
     "lint:js": "pnpm eslint-config '**/*.{ts,tsx,mjs,js}'",
     "lint:md": "pnpm eslint-config '**/*.{md,mdx}'",
     "lint:format": "pnpm prettier --check '**/*'",
-    "lint:styles": "pnpm stylelint '**/*.scss'",
+    "lint:scss": "pnpm stylelint '**/*.scss' --config .stylelintrc-scss.json",
+    "lint:css": "pnpm stylelint '**/*.css' --config .stylelintrc-css.mjs",
+    "lint:styles": "pnpm lint:scss && pnpm lint:css",
     "lint:fix": "pnpm lint:styles --fix && pnpm lint:js --fix && pnpm lint:md --fix && pnpm lint:format --write",
     "reset": "pnpm clean && pnpm install --force",
     "chromatic": "chromatic",
     "changeset": "changeset",
     "plop": "plop",
     "pkg:aio": "pnpm -F @kaizen/components",
+    "pkg:tokens": "pnpm -F @kaizen/design-tokens",
+    "pkg:tailwind": "pnpm -F @kaizen/tailwind",
+    "pkg:bundler": "pnpm -F @kaizen/package-bundler",
     "ci:version": "pnpm turbo build && pnpm changeset version",
     "ci:publish": "pnpm turbo build && pnpm changeset publish",
     "update-icons": "pnpm pkg:aio update-icons",
     "i18n:extract": "pnpm pkg:aio i18n:extract",
     "i18n:extract:remove-obsolete": "pnpm pkg:aio i18n:extract --deleteObsoleteTranslations"
   },
   "dependencies": {
-    "@kaizen/design-tokens": "workspace:*",
-    "@kaizen/tailwind": "workspace:*",
-    "@stylistic/eslint-plugin": "^2.6.2",
-    "classnames": "^2.5.1",
-    "identity-obj-proxy": "^3.0.0",
-    "lodash.isempty": "^4.4.0",
-    "normalize.css": "^8.0.1",
     "react": "^18.3.1",
-    "react-dom": "^18.3.1",
-    "react-highlight": "^0.15.0"
+    "react-dom": "^18.3.1"
   },
   "devDependencies": {
     "@changesets/cli": "^2.27.7",
     "@cultureamp/changelog-github": "^0.1.0",
-    "@storybook/addon-a11y": "^8.2.8",
-    "@storybook/addon-actions": "^8.2.8",
-    "@storybook/addon-backgrounds": "^8.2.8",
-    "@storybook/addon-essentials": "^8.2.8",
-    "@storybook/addon-interactions": "^8.2.8",
-    "@storybook/addon-links": "^8.2.8",
-    "@storybook/addon-webpack5-compiler-swc": "^1.0.5",
-    "@storybook/blocks": "^8.2.8",
-    "@storybook/components": "^8.2.8",
-    "@storybook/core-events": "^8.2.8",
-    "@storybook/manager-api": "^8.2.8",
-    "@storybook/react": "^8.2.8",
-    "@storybook/react-webpack5": "^8.2.8",
-    "@storybook/test": "^8.2.8",
-    "@storybook/test-runner": "^0.19.1",
-    "@storybook/theming": "^8.2.8",
-    "@tanstack/react-query": "^5.51.23",
+    "@storybook/addon-a11y": "^8.2.9",
+    "@storybook/addon-actions": "^8.2.9",
+    "@storybook/addon-backgrounds": "^8.2.9",
+    "@storybook/addon-essentials": "^8.2.9",
+    "@storybook/addon-interactions": "^8.2.9",
+    "@storybook/addon-links": "^8.2.9",
+    "@storybook/blocks": "^8.2.9",
+    "@storybook/components": "^8.2.9",
+    "@storybook/core-events": "^8.2.9",
+    "@storybook/react": "^8.2.9",
+    "@storybook/test": "^8.2.9",
+    "@storybook/theming": "^8.2.9",
+    "@stylistic/eslint-plugin": "^2.6.2",
```

Comment:
Can these be removed now that you've moved it into a workspace?

Reply:
> "@storybook/*": "Required for Storybook stories across all packages",